docs/Consent.md

```diff
@@ -0,0 +1,149 @@
+### Semantic model figure
+
+<p align="center">
+    <a href="../images/rdf/8_Consent.png" target="_blank">
+        <img src="../images/rdf/8_Consent.png">
+    </a>
+</p>
+
+
+***
+### Example rdf (turtle)
```

Comment:
I would suggest "Example RDF Data (Turtle)" as RDF (capital) is just a framework, though all of us in the CDE group know what it means here. Still, it would be great if we make it consistent with the outside world :) But this is future work that needs "CDE model people" to agree with,  and probably will be in the next PR.